dashboard/lib/tasks/seed.rake

```diff
@@ -461,6 +461,12 @@ namespace :seed do
     sh('mysqldump -u root -B dashboard_test > db/ui_test_data.sql')
   end
 
+  timed_task :import_pegasus_data do
+    params = {db: DASHBOARD_DB, table_prefix: "google_sheets_shared_"}
+    files_to_import = %w[data/cdo-languages.csv data/cdo-donors.csv]
+    files_to_import.each {|file_to_import| CsvToSqlTable.new(pegasus_dir(file_to_import), params).import}
+  end
+
```

Comment:
instead of calling this task from a cronjob, this task can just be added to the list of `FULL_SEED_TASKS` on the next line. that will make sure that dashboard tables get updated whenever we deploy.

Reply:
correction: it will need to be added to `UI_TEST_SEED_TASKS` as well.

Reply:
Done